shoop/core/pricing/__init__.py

```diff
@@ -155,16 +142,16 @@ def get_pricing_steps(self, context, product_id):
         """
         return [(0, TaxlessPrice(0))]
 
-    def get_prices(self, context, product_ids, quantity=1):
+    def get_prices(self, context, products, quantity=1):
```

Comment:
Ditto here; this should also be able to accept a list of product IDs or products.